.gitignore

```diff
@@ -2,6 +2,7 @@
 npm-debug.log
 node_modules/
 pvt/
+package-lock.json
```

Comment:
I just added it since it always mentions package-lock.json as untracked. Removed the line from .gitignore.